packages/components/psammead-podcast-promo/src/components/card-image.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { number, string } from 'prop-types';
+
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+import ImagePlaceholder from '@bbc/psammead-image-placeholder';
+
+const CardImage = styled.figure`
+  width: 88px;
+  display: inline-block;
+  vertical-align: top;
+  margin: 8px 0 0 8px;
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    width: 109px;
+  }
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    width: 178px;
+    margin: 0;
+  }
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    width: 100%;
```

Comment:
From the designs it looks like the row arrangement below 1008px could also be needed at 1008px and above? Maybe the screen width isn't appropriate as the only determinant for which arrangement of the card to use? 

<img width="526" alt="Screenshot 2020-11-20 at 16 38 32" src="https://user-images.githubusercontent.com/9042672/99825538-0d457a80-2b4f-11eb-976e-0365f79bb9a9.png">

OSS contributors may also be like 'ooh a podcast card coool wait WHAT-'
<img width="1141" alt="Screenshot 2020-11-20 at 16 34 25" src="https://user-images.githubusercontent.com/9042672/99826317-1d118e80-2b50-11eb-98ae-51e20263b168.png">

Reply:
aye it's definitely a bit janky in that regard - I'm not too sure how to solve this without using something like container queries

did consider adding some `compact` prop to make this configurable, but considering we're only going with this design for now, that's probably something that should wait until there's a v2 of this